src/amo/reducers/reviews.js

```diff
@@ -364,13 +384,13 @@ export default function reviewsReducer(
       return changeViewState({
         state,
         reviewId: action.payload.reviewId,
-        stateChange: { editingReview: true },
+        stateChange: { beginningToDeleteReview: false, editingReview: true },
       });
     case SHOW_REPLY_TO_REVIEW_FORM:
       return changeViewState({
         state,
         reviewId: action.payload.reviewId,
-        stateChange: { replyingToReview: true },
+        stateChange: { beginningToDeleteReview: false, replyingToReview: true },
```

Comment:
Just wondering, what's the use case for this? I couldn't seem to replicate a condition where I am deleting a review, but can also reply to it.

Reply:
In an earlier version the controls were still visible while you were deleting so it was possible back then.